earthfile2llb/interpreter.go

```diff
@@ -891,19 +891,19 @@ func (i *Interpreter) handleCopy(ctx context.Context, cmd spec.Command) error {
 	for index, src := range srcs {
 		var artifactSrc domain.Artifact
 		var parseErr error
-		if isInParansForm(src) {
+		if isInParamsForm(src) {
 			// COPY (<src> <flag-args>) ...
-			artifactStr, extraArgs, err := parseParans(src)
+			artifactStr, extraArgs, err := parseParams(src)
```

Comment:
I couldn't really tell if this was `parens` or `params`. I've taken a guess. I'm not claiming it's educated.

It's possible this is a term-of-art...